test/syntax/idl/constructor.widl

```diff
@@ -1,9 +1,9 @@
 // Extracted from http://dev.w3.org/2006/webapi/WebIDL/ on 2011-05-06
-[Constructor,
- Constructor(float radius)]
 interface Circle {
+  constructor();
+  constructor(float radius);
```

Comment:
Wonder if we should add our own cases too with other argument things... probably not necessary given everything uses the same code path.

Reply:
..e.g. variadic, optional, defaults, etc.

Reply:
I think it's not strictly needed but also does not harm. I'll add them as it's simple enough.